src/Illuminate/Http/Client/RequestException.php

```diff
@@ -36,7 +74,9 @@ protected function prepareMessage(Response $response)
     {
         $message = "HTTP request returned status code {$response->status()}";
 
-        $summary = Message::bodySummary($response->toPsrResponse());
+        $summary = static::$truncateAt
+            ? Message::bodySummary($response->toPsrResponse(), static::$truncateAt)
+            : Message::toString($response->toPsrResponse());
```

Comment:
Maybe, you want to do this:
```suggestion
        $psrResponse = $response->toPsrResponse();
        $summary = static::$truncateAt
            ? Message::bodySummary($psrResponse, static::$truncateAt)
            : Message::toString($psrResponse);
```

Reply:
I personally like the formatting of the way it is now, but of course can be modified by the team 👍